learning_assistant/api.py

```diff
@@ -112,3 +117,17 @@ def get_block_content(request, user_id, course_id, unit_usage_key):
         cache.set(cache_key, cache_data, getattr(settings, 'LEARNING_ASSISTANT_CACHE_TIMEOUT', 360))
 
     return cache_data['content_length'], cache_data['content_items']
+
+
+def render_prompt_template(request, user_id, course_id, unit_usage_key):
+    """
+    Return a rendered prompt template, specified by the LEARNING_ASSISTANT_PROMPT_TEMPLATE setting.
+    """
+    unit_content = ''
+    if unit_usage_key:
+        content_length, unit_content = get_block_content(request, user_id, course_id, unit_usage_key)
```

Comment:
Do we want to log anything about content length here? It might be a useful way to estimate how long each piece of content is.

Reply:
That's a good idea to get a sense of the length. I can see us needing that information in the future.